core/services/synchronization/presenters.go

```diff
@@ -30,7 +30,7 @@ func (p SyncJobRunPresenter) MarshalJSON() ([]byte, error) {
 		Status     string                 `json:"status"`
 		Error      null.String            `json:"error"`
 		CreatedAt  string                 `json:"createdAt"`
-		Amount     *assets.Link           `json:"amount"`
+		Payment    *assets.Link           `json:"payment"`
```

Comment:
Since the key is changing, can we update the `explorer` fixtures, even though they seem to be unused...

**Explorer:**
![image](https://user-images.githubusercontent.com/635121/64052850-13b49480-cb4e-11e9-8744-73a6880046ec.png)

https://github.com/smartcontractkit/chainlink/blob/develop/explorer/src/__tests__/fixtures/JobRun.fixture.json#L7
etc